accepted/00062-hub-xmlrpc-api.md

```diff
@@ -33,8 +33,11 @@ Content management APIs will work as of today. Inter-Server Synchronization (ISS
 
 ### New service and endpoint
 A new XMLRPC API endpoint will be created, implemented by a new service called the "XMLRPC Gateway API" (simply called "Gateway" from now on in this document). Technology-wise:
-  * Implementation will be in Python based on top of Tornado and [tornado-xmlrpc](https://pypi.org/project/tornado-xmlrpc/) library
-  * all I/O should be handled asynchronously via Tornado. Calls to several Servers should happen in parallel, with a maximum timeout value for unreachable/not responding Servers
+  * Implementation will be based on asynchronous I/O. Calls to several Servers should happen concurrently  
+  * The exact technology choice is delegated to the implementation phase and could be:
+    * typed Python, based on Tornado and [tornado-xmlrpc](https://pypi.org/project/tornado-xmlrpc/). Rationale: learning Tornado, which is also currently used in core Salt (rumored to be potentially replaced in future)
```

Comment:
I would suggest to put more full version of the rumour:
`(rumored to be potentially replaced with asyncio-based [trio](https://github.com/python-trio/trio) in future)`

Reply:
Addressed